clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStats.java

```diff
@@ -53,6 +53,7 @@ public class IngestionStats {
   protected static final String OFFSET_REGRESSION_DCR_ERROR = "offset_regression_dcr_error";
   protected static final String TOMBSTONE_CREATION_DCR = "tombstone_creation_dcr";
   protected static final String READY_TO_SERVE_WITH_RT_LAG_METRIC_NAME = "ready_to_serve_with_rt_lag";
+  public static final String VERSION_TOPIC_INGESTION_OFFSET_REWIND_COUNT = "vt_ingestion_task_offset_rewind_count";
```

Comment:
Is this the metric name? I don't think we need to put ingestion key word in it. This metric is used to alert Kafka data loss. Can we rename it to "vt_end_offset_rewind_count" or "version_topic_end_offset_rewind_count"?

Reply:
Thanks Min for the review. Sounds reasonable.